client/a8c-for-agencies/sections/sites/sites-dashboard/style.scss

```diff
@@ -461,4 +461,13 @@
 			}
 		}
 	}
+
+	.backup__page {
+		.daily-backup-status__restore-button,
+		.status-card__no-backup-last-backup,
+		.activity-card__toolbar--reverse,
+		.backup-date-picker {
+			display: none;
+		}
+	}
```

Comment:
We have to document/explain, why we are hiding this functionality. In fact, we shouldn't hide it while we are working on the Preview Pane tabs because it is an issue we have to fix, not to hide.

Reply:
The explanation is in this PR's description, those functionalities depend on using Jetpack Manage's URL as the base URL.